sdk/src/trace/batch_span_processor.cc

```diff
@@ -214,6 +218,7 @@ void BatchSpanProcessor::Export()
           nostd::span<std::unique_ptr<Recordable>>(spans_arr.data(), spans_arr.size()));
       NotifyCompletion(notify_force_flush, synchronization_data_);
     }
+#ifdef ENABLE_ASYNC_EXPORT
```

Comment:
Just like in `SimpleSpanProcessor`, `if (is_export_async_ == false)` should only be checked when `ENABLE_ASYNC_EXPORT` is defined, or we should always call sync Export.

Reply:
Done